sql/sp_head.cc

```diff
@@ -3114,6 +3114,7 @@ sp_instr_stmt::execute(THD *thd, uint *nextp)
   DBUG_ENTER("sp_instr_stmt::execute");
   DBUG_PRINT("info", ("command: %d", m_lex_keeper.sql_command()));
 
+  const bool enable_slow_log_backup = thd->enable_slow_log;
```

Comment:
Since this is not only a backup which is only restored later, but also used for the slow logging decision, it could use a different name, "outer_thd_enable_slow_log", or similar